source/js/src/even.js

```diff
@@ -111,7 +111,7 @@
 
     $(window).scroll(function () {
       var headerlinkTop = $.map($headerlink, function (link) {
-        return $(link).offset().top;
+        return $(window).scrollTop() + $(link).offset().top;
```

Comment:
Note that `$(window).scrollTop()` (as a getter) is more compatible than either `$(document.body).scrollTop()` or `$('html').scrollTop()`, since according to [this issue](https://bugs.chromium.org/p/chromium/issues/detail?id=766938) and [this comment](https://stackoverflow.com/questions/5371139/window-scrolltop-vs-document-scrolltop#comment-20034307) they won't work in Chrome.